docs/core/docker/build-container.md

```diff
@@ -0,0 +1,391 @@
+> * Create and configure a Dockerfile for .NET Core
+> * Build a Docker image
+> * Create and run a Docker container
+
+This tutorial teaches the Docker container build and deploy tasks for a .NET Core application. The *Docker platform* uses the *Docker engine* to quickly build and package apps as *Docker images*. These images are written in the *Dockerfile* format to be deployed and run in a layered container.
+
+## Prerequisites
+
+To complete this tutorial, install the following prerequisites:
+
+- [.NET Core 2.2 SDK](https://dotnet.microsoft.com/download)\
+If you have .NET Core installed, use the `dotnet --info` command to determine which SDK you're using.
+
+- [Docker Community Edition](https://www.docker.com/products/docker-desktop)
+
+- A temporary working directory for the *Dockerfile* and .NET Core example app.
+
+## Create .NET Core app
+
+You need a .NET Core app that the Docker container will run. Open your terminal, create a working directory, and enter it. In the working directory, run the following command to create a new project in a subdirectory named app:
+
+```console
+dotnet new console -o app -n myapp
+```
+
+That command creates a new directory named *app* and generates a "Hello World" app. You can test this app to see what it does. Enter the *app* directory and execute the command `dotnet run`. You'll see the following output:
+
+```console
+> dotnet run
+Hello World!
+```
+
+The default template creates an app that prints to the terminal and then exits. For this tutorial, you'll use an app that loops indefinitely. Open the **Program.cs** file in a text editor. It should currently look like the following code:
+
+```csharp
+using System;
+
+namespace myapp
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hello World!");
+        }
+    }
+}
+```
+
+Replace the file with the following code that counts numbers every second:
+
+```csharp
+using System;
+
+namespace myapp
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var counter = 0;
+            var max = args.Length != 0 ? Convert.ToInt32(args[0]) : -1;
+            while(max == -1 || counter < max)
+            {
+                counter++;
+                Console.WriteLine($"Counter: {counter}");
+                System.Threading.Tasks.Task.Delay(1000).Wait();
+            }
+        }
+    }
+}
+```
+
+Save the file and test the program again with `dotnet run`. Remember that this app runs indefinitely. Use the cancel command <kbd>CTRL + C</kbd> to stop it. You'll see the following output:
+
+```console
+> dotnet run
+Counter: 1
+Counter: 2
+Counter: 3
+Counter: 4
+^C
+```
+
+If you pass a number on the command line to the app, it will only count up to that amount and then exit. Try it with `dotnet run -- 5` to count to five.
+
+> [!NOTE]
+> Any parameters after `--` are passed to your application.
+
+### Using a newer SDK
```

Comment:
Suggestion: move this section above, right after pre-requisites. The `dotnet --info` will tell the reader what SDK they are using, and if a `global.json` is determining the current SDK selection. It might flow better.

```suggestion
### Use SDK version 2.2
```